LayoutTests/imported/w3c/web-platform-tests/css/selectors/w3c-import.log

```diff
@@ -219,11 +219,8 @@ List of files:
 /LayoutTests/imported/w3c/web-platform-tests/css/selectors/historical-xmlid-expected.xht
 /LayoutTests/imported/w3c/web-platform-tests/css/selectors/historical-xmlid.xht
 /LayoutTests/imported/w3c/web-platform-tests/css/selectors/hover-002.html
-/LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-default-ns-001-expected.html
 /LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-default-ns-001.html
-/LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-default-ns-002-expected.html
 /LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-default-ns-002.html
-/LayoutTests/imported/w3c/web-platform-tests/css/selectors/is-default-ns-003-expected.html
```

Comment:
Not sure how you ended up deleting these files, but it seems wrong to me, the test wouldn't be able to run. Fwiw, I resynced this directory very recently: https://github.com/WebKit/WebKit/pull/14420

Reply:
Yeah it's wrong. Filed https://bugs.webkit.org/show_bug.cgi?id=257674.